addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -814,6 +814,6 @@ a.no-decoration {
     }
 }
 
-.accordion-button:not(.collapsed).bg-transparent::after {
+.wsale_products_categories_list .accordion-button:not(.collapsed).bg-transparent::after {
     background-image: escape-svg($accordion-button-icon);
```

Comment:
> that rule doesn't work (the icon doesn't show)

In fact, this icon is displayed behind the UI icon as a background image  😬 👇 
<img width="794" alt="Capture d’écran 2024-08-08 à 16 58 30" src="https://github.com/user-attachments/assets/88a0b497-89e1-45e3-a4b4-a10a390b0a04">

This adaptation allows to scope only the related module and avoids to break other accordions.

Reply:
Sorry, my message wasn't clear. I understood the reason behind your fix, I'm just pointing out that their rule doesn't work in the first place. Unless I'm completely wrong, it's supposed to be there when certain options are toggled on:
![image](https://github.com/user-attachments/assets/2f19d5f7-d529-4f99-a277-eb9571c3098f)

But since the pseudo-element doesn't have any width/height, it's not shown.

So my comment wasn't about this PR, just a way to point the problem. But I should have done that in their channel. ;)